docker-plugin/src/main/java/io/micronaut/gradle/docker/NativeImageDockerfile.java

```diff
@@ -52,11 +52,17 @@
  */
 public abstract class NativeImageDockerfile extends Dockerfile implements DockerBuildOptions {
 
+    /**
+     * Main class for Provided runtime and Application Type: Micronaut Application.
+     */
+    public static final String MICRONAUT_LAMBDA_RUNTIME = "io.micronaut.function.aws.runtime.MicronautLambdaRuntime";
```

Comment:
I _think_ it would make more sense to have this in `MicronautDockerPlugin` or `DockerBuildStrategy`: this is not directly related to docker _files_, wdyt?